### DISCUSSION TITLE
Gathering questions about the System Requirement.

### DISCUSSION THREAD

**Discussion by ebrarkiziloglu** [bounswe/bounswe2024group5#72] [Ideas]
Let's identify our questions during requirement writing process.
Every team member should determine questions for their subcategory in the requirements.

**Comment by ebrarkiziloglu** [comment]
@odenizddd 
@HutkuC 
@semihYILMAZ37 
@set120120 
@SouthAscend 
@Ultiminati 
@ramazanoacar 
@arincdemir

**Comment by arincdemir** [comment]
Will the main search functionality only search the web sources(wikidata etc.) or will it also show related posts from other users? If it won't show other users' posts, how will one come across other users' posts when they are not following them?

**Comment by set120120** [comment]
- If the main algorithm search also displays posts, will it show posts from users who have been blocked or restricted by the user, but their posts are closely related and highly relevant to the search topic?
- Will the main search algorithm show the user only web sources and posts, or also comments on posts?

**Comment by SouthAscend** [comment]
- Will there be any sorting algorithm (view count, like count, newest to oldest, oldest to newest) on the feed page?
- Will there be any filtering (by topics, etc.) on the feed page?
- Will there be any posts from unfollowed users on the feed page? (Similar to the explore page's post, or "you might like these" posts.)

**Comment by ebrarkiziloglu** [comment]
About Posting:
* Are users expected to like AND dislike posts or is just liking enough?
* Should users be able to comment on posts?

**Comment by ramazanoacar** [comment]
- will hashtags exist in our app and if yes can a user go over different posts with same hashtags?
- what will be the options for signing up to our app? can users use an email or can they sign up with phones, facebook, gmail etc. also do we need to store personal data or any kind of data about our domain from the user?

**Comment by set120120** [comment]
- Can users search other users' profile?
- Can users use only semantic search or both semantic search and basic text search?
- If the search result is not good enough to show to the user, will similar search options be suggested to the user?
